Jakefile.js

```diff
@@ -4,7 +4,6 @@ var fs = require("fs");
 var os = require("os");
 var path = require("path");
 var child_process = require("child_process");
-var Linter = require("tslint");

```

Comment:
I'd like that. Would also stop me from randomly breaking one of them when I
forget to duplicate effort.

On Mon, Aug 15, 2016, 9:53 AM Nathan Shively-Sanders <
notifications@github.com> wrote:

> In Jakefile.js
> <https://github.com/Microsoft/TypeScript/pull/10313#discussion_r74793060>:
>
> > @@ -4,7 +4,6 @@ var fs = require("fs");
> >  var os = require("os");
> >  var path = require("path");
> >  var child_process = require("child_process");
> > -var Linter = require("tslint");
>
> we should think about switching Travis over to gulp so we don't have to
> duplicate effort like this in the future.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Microsoft/TypeScript/pull/10313/files/949bbf4e343afe9080698cc95bcf1dcf423478a0#r74793060>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ACzAMv6kXQAsxbA8_0eg5PQCd9hCZyyVks5qgJmOgaJpZM4Jjj11>
> .
>